### ISSUE TITLE
Add leetcode scraper

### ISSUE THREAD

**Issue by akstron** [ContriHUB/E-CP#4]
Add a web scraper to extract leetcode problem sample input/output.

**Comment by utkarsh1236** [comment]
Hi @akstron ! I had a query regarding this issue. LeetCode has a different format of displaying sample test cases and of accepting solutions since it requires you to submit your solution in a class. Then what would be the right way to store the sample input and output as we don't really know how the input is being passed into the solution?

**Comment by utkarsh1236** [comment]
For example, in this [problem](https://leetcode.com/problems/continuous-subarray-sum/) the first example states that nums = [23,2,4,6,7], k = 6 but I don't believe we can just store them as it is in the input file.

**Comment by akstron** [comment]
That's true. We can think about parsing the content before storing it in the test file. For now, we can create an array parser, and a 2d array parser.
